### PR TITLE
Adrianbarbulescu branch

### PR DESCRIPTION
**Pull request by ST4NSB**
“added adrian barbulescu tasks”

### CODE REVIEW

File: adrianbarbulescu-dir/adrianbarbulescu.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+this file was created by adrian barbulescu
```
Comment (andreio16) [inline adrianbarbulescu-dir/adrianbarbulescu.txt]: please make "adrian barbulescu" as uppercase

Comment (ST4NSB) [reply adrianbarbulescu-dir/adrianbarbulescu.txt]: am schimbat, e bn asa $efu?

File: adrianbarbulescu-dir/adrianbarbulescu.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
+this file was created by Adrian Barbulescu
+2nd commit
```
Comment (mirceacs) [inline adrianbarbulescu-dir/adrianbarbulescu.txt:2]: God job, do a better one

**Review by mirceacs — CHANGES_REQUESTED** [review]

**Review by andreio16 — APPROVED** [review]

File: adrianbarbulescu-dir/adrianbarbulescu.txt (hunk 2 above)
Comment (ST4NSB) [reply adrianbarbulescu-dir/adrianbarbulescu.txt:2]: what a ****

**Review by mirceacs — APPROVED** [review]

**Comment by ST4NSB** [comment]
Merged into master I quess

**Pull request merged**